package.json

```diff
@@ -8,7 +8,15 @@
     "build": "webpack -p --env=production",
     "serv": "babel-node server.js --presets es2015,stage-2",
     "lint": "eslint ./src/js/**/*.js",
-    "test": "jest"
+    "test": "jest test --all --updateSnapshot --no-cache",
```

Comment:
I'm a bit concerned about updating snapshots with test script. The problem I see is: we made some changes and modified structure of any component, which we didn't intend to do, then during the pipeline snapshot will be updated and test will pass. even though the tree has changed. So our test task becomes partially useless. What do you think about it?